src/services/aws/pricing/function/WriteProductsToFile.mocks.ts

```diff
@@ -1,565 +1,582 @@
+        vcpu: "2",
+        classicnetworkingsupport: "false",
+        capacitystatus: "UnusedCapacityReservation",
+        locationType: "AWS Region",
+        storage: "EBS only",
+        instanceFamily: "General purpose",
+        operatingSystem: "Linux",
+        intelAvx2Available: "Yes",
+        regionCode: "eu-central-1",
+        physicalProcessor: "Intel Skylake E5 2686 v5",
+        clockSpeed: "3.1 GHz",
+        ecu: "Variable",
+        networkPerformance: "Up to 5 Gigabit",
+        servicename: "Amazon Elastic Compute Cloud",
+        instancesku: "Q7XFK558YJTFPWDH",
+        vpcnetworkingsupport: "true",
+        instanceType: "t3.micro",
+        tenancy: "Dedicated",
+        usagetype: "EUC1-UnusedDed:t3.micro",
+        normalizationSizeFactor: "0.5",
+        intelAvxAvailable: "Yes",
+        processorFeatures:
+          "AVX; AVX2; Intel AVX; Intel AVX2; Intel AVX512; Intel Turbo",
+        servicecode: "AmazonEC2",
+        licenseModel: "No License required",
+        currentGeneration: "Yes",
+        preInstalledSw: "SQL Web",
+        location: "EU (Frankfurt)",
+        processorArchitecture: "64-bit",
+        marketoption: "OnDemand",
+        operation: "RunInstances:0200",
+        availabilityzone: "NA",
       },
-      "serviceCode": "AmazonEC2",
-      "terms": {
-        "OnDemand": {
-          "Y6JK4U8NZRJW6Z2M.JRTCKXETXF": {
-            "priceDimensions": {
-              "Y6JK4U8NZRJW6Z2M.JRTCKXETXF.6YS6EN2CT7": {
-                "unit": "Hrs",
-                "endRange": "Inf",
-                "description": "$0.0804 per Dedicated Unused Reservation Linux with SQL Web t3.micro Instance Hour",
-                "appliesTo": [],
-                "rateCode": "Y6JK4U8NZRJW6Z2M.JRTCKXETXF.6YS6EN2CT7",
-                "beginRange": "0",
-                "pricePerUnit": {
-                  "USD": "0.0804000000"
-                }
-              }
+      sku: "Y6JK4U8NZRJW6Z2M",
+    },
+    serviceCode: "AmazonEC2",
+    terms: {
+      OnDemand: {
+        "Y6JK4U8NZRJW6Z2M.JRTCKXETXF": {
+          priceDimensions: {
+            "Y6JK4U8NZRJW6Z2M.JRTCKXETXF.6YS6EN2CT7": {
+              unit: "Hrs",
+              endRange: "Inf",
+              description:
+                "$0.0804 per Dedicated Unused Reservation Linux with SQL Web t3.micro Instance Hour",
+              appliesTo: [],
+              rateCode: "Y6JK4U8NZRJW6Z2M.JRTCKXETXF.6YS6EN2CT7",
+              beginRange: "0",
+              pricePerUnit: {
+                USD: "0.0804000000",
+              },
             },
-            "sku": "Y6JK4U8NZRJW6Z2M",
-            "effectiveDate": "2022-10-01T00:00:00Z",
-            "offerTermCode": "JRTCKXETXF",
-            "termAttributes": {}
-          }
-        }
+          },
+          sku: "Y6JK4U8NZRJW6Z2M",
+          effectiveDate: "2022-10-01T00:00:00Z",
+          offerTermCode: "JRTCKXETXF",
+          termAttributes: {},
+        },
```

Comment:
why is there a comma following the closing bracket of the object?

Reply:
Same thing on Line 73

Reply:
And in other places

Reply:
This is a snippet of pricing data returned from the pricing API unedited. The comma at the end has no effect, it's more a convention to have or have not but it shouldn't be inconsistent as it is in this file. As it's just a mock we can leave it as is